rootfs/scheduler/__init__.py

```diff
@@ -604,15 +604,11 @@ def _create_rc(self, name, image, command, **kwargs):  # noqa
 
         # Check if it is a slug builder image.
         # Example format: golden-earrings:git-5450cbcdaaf9afe6fadd219c94ac9c449bd62413s
-        image_name = image[image.index(':')+1:]
-        if '-' in image_name:
-            vcs, sha = image_name.split('-')
-            if vcs == 'git' and len(sha) in [8, 40] and app_type == 'web':
-                imgurl = 'http://{}/git/home/{}/push/slug.tgz'.format(
-                    settings.S3EP,
-                    image
-                )
-                TEMPLATE = RCB_TEMPLATE
+        if kwargs.get('build_type', {}) == "buildpack":
```

Comment:
Only worry about the defaulting to `{}` here since we are using strings in there. Maybe no defaulting and rather have things blow up hard if somehow `app_build_type` doesn't return anything, not that I see how it would :)